docs/inclusion-proofs.md

```diff
@@ -1,8 +1,8 @@
-# QGB Fraud Proofs
+# BlobStream Fraud Proofs
 
-## Celestiums Intro
+## BlobStream Intro
 
-A Celestium is a blockchain that uses Celestia for data availability but settles on any EVM chain. The QGB operates by having the Celestia validator set periodically sign over batched data commitments and validator set updates, which are relayed an EVM smart contract. The data commitments are stored in the EVM chain's state, and can be used to prove inclusion of any data historically posted to Celestia.
+A BlobStream rollup is a blockchain that uses Celestia for data availability but settles on any EVM chain. The BlobStream operates by having the Celestia validator set periodically sign over batched data commitments and validator set updates, which are relayed an EVM smart contract. The data commitments are stored in the EVM chain's state, and can be used to prove inclusion of any data historically posted to Celestia.
```

Comment:
[note to reviewers]
I changes `Celestium` to `BlobStream rollup`, not sure if that the right name or there is another name that's being decided upon